cmd/main.go

```diff
@@ -61,6 +62,7 @@ func init() {
 const deployment_type_k8s = "k8s"
 const deployment_type_onprem = "onprem"
 const deployment_type_ocp = "ocp"
+const listSeparator = ","
```

Comment:
Looks like default configuration, can probably be inside the package.

Reply:
I'll see if I can move it.